### PR TITLE
Changed repository prefix from SQL to Eloquent

### LINKED ISSUE #45: Repository naming
Currently we use an `SQL` prefix to name the repositories in #30. While this is good, I think `Eloquent` would be more appropriate and correct since the repositories are implemented through `Eloquent`.

@tenorz007 @SkaeX what do you think?

### PR DESCRIPTION
**Pull request by BrainMaestro**
Fixes #45

### CODE REVIEW

**Comment by BrainMaestro** [comment]
Approved

File: tests/unit/Repositories/Database/EloquentQuestionRepositoryTest.php [hunk 1]
```diff
@@ -41,7 +41,7 @@ public function testGetQuestionById()
     {
         $question = self::$questionRepository->getQuestionById($this->question->id);
 
-        $this->assertEquals(SQLQuestionRepository::transform($this->question), $question);
+        $this->assertEquals(EloquenyQuestionRepository::transform($this->question), $question);
```
Comment (SkaeX) [inline tests/unit/Repositories/Database/EloquentQuestionRepositoryTest.php]: Typo

Comment (BrainMaestro) [inline tests/unit/Repositories/Database/EloquentQuestionRepositoryTest.php]: Yes typo. Fixing now. Good spot.

**Comment by SkaeX** [comment]
I think we are good here

**Comment by SkaeX** [comment]
Approved

**Pull request merged**